movie_REST_API/movie_REST_API/Service/CoreDataService/CoreDataServiceProtocol.swift

```diff
@@ -0,0 +1,10 @@
+// CoreDataServiceProtocol.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import Foundation
+
+/// Протокол работы с CoreData
+protocol CoreDataServiceProtocol {
+    func saveMovieCoreData(type: TypeMovie, movies: [Movie])
+    func getMovieCoreData(type: TypeMovie) -> [Movie]?
```

Comment:
```suggestion
    func saveMovieCoreData(type: TypeMovie, movies: [Movie])
    func getMovieCoreData(type: TypeMovie) -> [Movie]?
```
сюда еще 2 публичных метода наверное перенести надо